### ISSUE TITLE
How to deal with images with various size in VOC

### ISSUE THREAD

**Issue by BingzheWu** [fabianbormann/Tensorflow_Segmentation#1]
Hi, I have saw that in this code you use the placeholder with fixed shape?  I really want to know
how to deal with images with different size in semantic segmentation.

**Comment by fabianbormann** [comment]
Hey, yes currently I use a hard coded pixel size but in the future I want to use a variable shape as well. So I think (if the unpool layer works as expected) I will later also replace the line `image = tf.placeholder(tf.float32, shape=[224, 224, 3])` by `image = tf.placeholder(tf.float32, shape=[None, None, 3])` -> but it's important that the width and height is greater than 244 because of the pooling layer and the filter size (7x7) of fc_6.

**Comment by fabianbormann** [comment]
I would love to see your PR if you want start to contribute 😊

**Comment by varad2512** [comment]
Does the net actually work?

**Comment by fabianbormann** [comment]
@varad2512 no currently [I can not train the net](https://github.com/fabianbormann/Tensorflow-DeconvNet-Segmentation/blob/master/DeconvNet.py#L50), because I ran out of memory on a GTX960 at home so I need to test it in university. This week I will finish my exams so I can start testing and implementing next week.

**Comment by varad2512** [comment]
Hi Fabian,

Yeah okay.
*Once you get any results please could you send an update? *

Thanks,
Varad.

**Comment by fabianbormann** [comment]
@varad2512 yeah of course and I will also update the readme ;)

**Comment by Yaffa1607** [comment]
Thanks for the net. I'm looking forward for the trained net

**Comment by BingzheWu** [comment]
I am trying to transform the caffemodel of fcn to tensorflow model. All goes right except the the deconvolutional layers. And I have found that there is some little difference in the implementation of pooling  between caffe and tensorflow ( I have solved this problem through some tricks). I am very interesting on implementing some seg models by tensorflow. So can I join you and strat to contribute on 
this subject?
